### ISSUE TITLE
Get PO for WGBS sample sequencing (15 samples)

### ISSUE THREAD

**Issue by sr320** [urol-e5/deep-dive-expression#19]
quote 
[Genohub_Quote_54759877_Accepted (2).pdf](https://github.com/user-attachments/files/18130905/Genohub_Quote_54759877_Accepted.2.pdf)

**Comment by kubu4** [comment]
PO request submitted.

**Comment by sr320** [comment]
any word on this?

**Comment by kubu4** [comment]
Yeah, it might be awhile. Genohub isn't registered as a supplier with UW, so I probably can't get a PO until that happens. Just this morning I requested a link to instructions for suppliers to get set up with UW. I'll pass that info to Genohub when get it.

**Comment by kubu4** [comment]
Got link to add Genohub as supplier to UW system yesterday. Genohub should've received email with instructions on how to register with UW.

If i haven't heard anything, I'll follow up with Genohub later this week and/or early next week.

**Comment by kubu4** [comment]
I've just sent Genohub an email to check on their registration status. Will update when I hear back.

**Comment by kubu4** [comment]
Genohub is now an official supplier in the UW system.

PO request has been re-submitted.

**Comment by kubu4** [comment]
Received PO.

Will email Genohub with PO to complete transaction and initiate sequencing!
